app/app_aes.c

```diff
@@ -191,6 +190,17 @@ int app_aes_handler(ACVP_TEST_CASE *test_case) {
             rv = 1;
             goto err;
         }
+        switch (tc->tw_mode) {
+        case ACVP_SYM_CIPH_TWEAK_HEX:
+            iv = tc->iv;
+            break;
+        case ACVP_SYM_CIPH_TWEAK_NUM:
+        case ACVP_SYM_CIPH_TWEAK_NONE:
+        default:
+            printf("\nUnsupported tweak mode %d %d\n", tc->seq_num, tc->tw_mode);
+            rv = 1;
+            goto err;
```

Comment:
Coverity may complain about lack of break even after goto

Reply:
Fixed